addons/hw_drivers/controllers/driver.py

```diff
@@ -1,432 +1,398 @@
 #!/usr/bin/python3
 import logging
 import time
-from threading import Thread
-import usb
-import gatt
+from threading import Thread, Event
+from usb import core
+from gatt import DeviceManager as Gatt_DeviceManager
 import subprocess
-import netifaces as ni
+import netifaces
 import json
-import re
-from odoo import http
+from re import sub
+from odoo import http, _
 import urllib3
 from odoo.http import request as httprequest
```

Comment:
you already import odoo.http, so I think it is better to remove this extra import and use http.request in the code?